### ISSUE TITLE
Non-English DO / DON'T content is out of sync

### ISSUE THREAD

**Issue by emersonthis** [flattenthecurve/guide#295]
This should probably be broken into smaller issues to deal with language versions individually, but this should be done by someone more familiar with the translation process (than me). Also, it doesn't make sense for this work to happen right now, because we're in the process of [reorganizing the DO & DON'T content](https://github.com/flattenthecurve/guide/issues/294). 

This issue is basically a placeholder to make sure we loop back to this after #294 is closed.

Notice the different structures...

<img width="1587" alt="Screen Shot 2020-04-04 at 11 05 40 AM" src="https://user-images.githubusercontent.com/1728139/78459162-00c9eb80-766c-11ea-8331-b61fdcf9e2c8.png">
<img width="1587" alt="Screen Shot 2020-04-04 at 11 05 51 AM" src="https://user-images.githubusercontent.com/1728139/78459165-04f60900-766c-11ea-921f-658020791536.png">
<img width="1587" alt="Screen Shot 2020-04-04 at 11 06 12 AM" src="https://user-images.githubusercontent.com/1728139/78459170-0c1d1700-766c-11ea-85bd-7c6caa81119e.png">

**Comment by rousik** [comment]
I have found that page-styling and layout leaks into the translation quite significantly and we've had several rounds of non-english content falling behind and being buggy (e.g. when headers were reorganized or new home page design was rolled out). I wonder whether we either 1. can somehow provide better insulation between styling and content or 2. ensure that we can have some process for revisiting/updating non-english translations periodically.

What we might automate would be detection of styling/markdown discrepancies between english and non-english content snippets which would indicate that things need to be corrected.

**Comment by rousik** [comment]
What might make sense is to keep POC person for each language and fire off "please sync translation" requests when we reach significant overhauls or when language lags behind to a considerable degree so as to prevent organic bit-rot.

**Comment by emersonthis** [comment]
Thanks for starting this conversation! I’ve been thinking about this a lot and wanting to help make it smoother, but I don’t known much about the automation related to translations... 

If I understand correctly, what you said about the styles etc leaking into the content / translations is at the heart of this issue. Sounds like we either need to work some magic to work around the styles, or maybe rethink the design/implementation to accommodate this constraint.

**Comment by rousik** [comment]
I have opened #368 which tracks the effort to get all live languages in sync after the act_and_prepare reorg. I'm closing this as a duplicate.

**Comment by rousik** [comment]
This issue is a duplicate of #368
